src/pages/blog/the-end-of-password-pain-building-frictionless-authentication-at-the-guardian.md

```diff
@@ -0,0 +1,155 @@
+
+By analysing our reader creation journeys, we found that the need to choose a password prevents around 20% of readers from creating an account via the traditional email and password method, making it a significant point of friction in the account creation process. Tangentially, this can cause problems, where the reader might end up signed in on another device or browser rather than the one they were trying to use if they clicked on the link on another device These issues prevent our readers from accessing our products in scenarios where account creation is mandatory. Furthermore, the majority of identity-related complaints to our User Help and Customer Experience teams revolve around passwords.
+
+We’ve responded to this by now making passwords optional when it comes to sign in and account creation. Instead, by default, we now ask the reader to verify their identity by asking them to submit a short-lived one-time use password (OTP), sometimes called a “passcode”, sent to their email inbox as part of our sign in, account creation, and password reset journeys. Using passcodes also eliminates the need to send readers a link for verification purposes, which improves the user experience in those scenarios too.
+
+## Account creation - why we decided to use codes instead of links
+
+While not related to security specifically, we used to ensure that for account creation, a new reader would type in their email address and we would send a verification link to that email in order for them to verify their account, set a password, and get signed in.
+
+However we found for some readers this was a huge cause of frustration, and was affecting account creation completion rates. The common issues we found readers encountering were:
+
+- Readers accessing their email on a different device
+
+  After starting the account creation process on one device, they would have to complete account creation on another, and then sign in again on the original device.
+
+- Around 20% of readers were ending up signed in on theguardian.com in their device browser and not in our news app
+
+  When creating an account via one of our apps, we can’t always guarantee that the verification link will take the reader back to the app. This is because deep links can be overridden by device settings across both iOS and Android. It means that high proportions of readers were ending up signed-in on theguardian.com in their device browser, rather than in the app, which was particularly excruciating for the new Feast app. This was mitigated by using an [app specific completion page](https://github.com/guardian/gateway/pull/2580) asking the reader to go back to the app and sign in again, but this wasn’t ideal.
```

Comment:
Past tense perhaps?
```suggestion
  When creating an account via one of our apps, we can’t always guarantee that the verification link will take the reader back to the app. This is because deep links can be overridden by device settings across both iOS and Android. It meant that high proportions of readers were ending up signed-in on theguardian.com in their device browser, rather than in the app, which was particularly excruciating for the new Feast app. This was mitigated by using an [app specific completion page](https://github.com/guardian/gateway/pull/2580) asking the reader to go back to the app and sign in again, but this wasn’t ideal.
```